src/helperFunctions/plugin.py

```diff
@@ -33,8 +37,11 @@ def _import_plugins(plugin_type):
         module_name = str(plugin_file).replace('/', '.')[len(src_dir + '/') : -len('.py')]
 
         loader = SourceFileLoader(module_name, str(plugin_file))
-        spec = importlib.util.spec_from_loader(loader.name, loader)
-        plugin_module = importlib.util.module_from_spec(spec)
+        spec = spec_from_loader(loader.name, loader)
+        if spec is None:
```

Comment:
Why and when can this happen?

Reply:
I'm not actually sure. Mypy claimed that `spec_from_loader()` may return `None` and I took a look in the code but wasn't really sure so I kept it.

Reply:
I wouldn't trust mypy here.

Reply:
But do you know for sure that it isn't the case? Better safe than sorry? :sweat_smile:

Reply:
Checking for error conditions where there are none documented and we don't have any test case to produce any error does not make sense imo.
So I'd better be sorry.

Reply:
I removed the check and added a `# type: ignore` instead